docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+|Authentication| TODO | `Area:Authentication` |
+|Custom Adapters| TODO | `Area:Custom Adapters` |
+|Docs| TODO | `Area:Docs` |
+|Functional Tests| TODO | `Area:Functional Tests` |
+|LG| TODO | `Area:LG` |
+|Samples| TODO | `Area:Samples` |
+|Schema| TODO | `Area:Schema` |
+|Skills| TODO | `Area:Skills` |
+|Streaming| TODO | `Area:Streaming` |
+|Teams| TODO | `Area:Teams` |
+|Telemetry| TODO | `Area:Telemetry` |
+|Testing Framework| TODO | `Area:Testing Framework` |
+
+### Priority
+
+Describes the priority of the issue. This label is required for any issue that is in scope for an iteration. High priority issue will be addressed first. All the issues with `Status:Approved` should have at least one of these labels before they can be worked on.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|P:0| Must Fix.  Release-blocker | ![#ee0701](https://via.placeholder.com/15/ee0701/000000?text=+) `#ee0701` | `P:0` |
+|P:1| Painful if we don't fix, won't block releasing | ![#ff8c00](https://via.placeholder.com/15/FF8C00/000000?text=+) `#ff8c00` | `P:1` |
+|P:2| Nice to have | ![#ffff00](https://via.placeholder.com/15/ffff00/000000?text=+) `#ffff00` | `P:2` |
+|P:3| Won't fix | ![#bfd4f2](https://via.placeholder.com/15/bfd4f2/000000?text=+) `#bfd4f2` | `P:3` |
+
+### Status
+
+Use these labels for providing additional information on the status of the issue.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|Draft| The issue definition is still being worked on and it is not ready to start development. Once the issue is ready the status should be changed to `Approved` or `Backlog`.| ![#ededed](https://via.placeholder.com/15/ededed/000000?text=+) `#ededed` | `Status:Draft` |
+|New| The issue has just been created and it has not been reviewed by the team. Once the issue is reviewed the status be changed to `Approved`, `Backlog`, `Needs information` or just closed.| ![#bfd4f2](https://via.placeholder.com/15/bfd4f2/000000?text=+) `#bfd4f2` | `Status:New` |
+|Needs information| The issue as described is incomplete or not well understood. It is waiting for further information before it can continue.| ![#ff8c00](https://via.placeholder.com/15/ff8c00/000000?text=+) `#ff8c00` | `Status:Needs information` |
+|Backlog| The issue is out of scope for the current iteration but it will be evaluated in a future release. | ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Status:Backlog` |
+|Approved| The issue has been reviewed and is ready to start working on it, it will be added to the work queue in the current iteration. | ![#0e8a16](https://via.placeholder.com/15/0e8a16/000000?text=+) `#0e8a16` | `Status:Approved` |
+|Blocked| Current progress is blocked on something else. | ![#ff8c00](https://via.placeholder.com/15/FF8C00/000000?text=+) `#ff8c00` | `Status:Blocked` |
```

Comment:
Would it be useful to have a couple different subcategories here for work that's blocked by another team (such as Teams), waiting on a finalized spec or schema, waiting on stakeholder sign-off, and so on?

Reply:
I'd rather have less categories, blocked means blocked and can't continue, the issue history should explain why.